ckanext/datalocale/public/css/datalocale.css

```diff
@@ -55,20 +55,65 @@
 
 .masthead .blue{
   background-color: #25B8EA;
+  -webkit-transition: color 1s, background-color 1s, box-shadow 1s;
+  transition: color 1s, background-color 1s, box-shadow 1s;
 }
 
 .masthead .orange{
   background-color: #EE4524;
+  -webkit-transition: color 1s, background-color 1s, box-shadow 1s;
+  transition: color 1s, background-color 1s, box-shadow 1s;
 }
 
 .masthead .grey{
   background-color: #b3b3b3;
+  -webkit-transition: color 1s, background-color 1s, box-shadow 1s;
+  transition: color 1s, background-color 1s, box-shadow 1s;
 }
 
 .masthead .green{
   background-color: #31b44d;
+  -webkit-transition: color 1s, background-color 1s, box-shadow 1s;
+  transition: color 1s, background-color 1s, box-shadow 1s;
```

Comment:
looks like you might benefit from an extra css class for button, so that you can factor out the transition setting.
```
<div class="masthead">
...
<li class="mainmenu orange">
...
```

and then
```
.masthead .mainmenu {
  -webkit-transition: color 1s, background-color 1s, box-shadow 1s;
  transition: color 1s, background-color 1s, box-shadow 1s;
}
```